### PR TITLE
PrestaShop Developer Conference 2023

### PR DESCRIPTION
**Pull request by kpodemski**

### CODE REVIEW

File: content/news/posts/2023/prestashop-developer-conference-2023.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title:  "PrestaShop Developer Conference in Paris : synopsis of the day"
```
Comment (coffrep) [inline content/news/posts/2023/prestashop-developer-conference-2023.md]:
```suggestion
title:  "PrestaShop Developer Conference in Paris: Synopsis Of The Day"
```

File: content/news/posts/2023/prestashop-developer-conference-2023.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+After welcoming attendees and Mathieu's introductory talk, the event truly kicked off with some outstanding presentations from the PrestaShop community. The day featured nearly 20 presentations by 25 speakers, many of whom traveled from distant locations to the Paris area to discuss PrestaShop.
+I won’t be going into too much detail about the presentations as you will be able to read about them and even watch them in a video format in the next few weeks. Speakers are currently preparing blog posts on this Build blog and three video presentations are currently planned on Live Update, with more to follow.
```
Comment (coffrep) [inline content/news/posts/2023/prestashop-developer-conference-2023.md]:
```suggestion
I won’t be going into too much detail about the presentations as you will be able to read about them and even watch them in a video format in the next few weeks. Speakers are currently preparing blog posts on this Build blog and three video presentations are visible or planned on [Live Update](https://www.youtube.com/@prestashopproject), with more to follow. 
```

File: content/news/posts/2023/prestashop-developer-conference-2023.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+After welcoming attendees and Mathieu's introductory talk, the event truly kicked off with some outstanding presentations from the PrestaShop community. The day featured nearly 20 presentations by 25 speakers, many of whom traveled from distant locations to the Paris area to discuss PrestaShop.
+I won’t be going into too much detail about the presentations as you will be able to read about them and even watch them in a video format in the next few weeks. Speakers are currently preparing blog posts on this Build blog and three video presentations are currently planned on Live Update, with more to follow. 
+
+<img src="/assets/images/2023/12/psdevcon1.jpg" style="height:400px;  margin: 20px auto;">
+
+During the day, presentations covered a wide array of topics, ranging from module development to new features of PrestaShop. The event also highlighted developer solutions, security concerns, and CI/CD deployment strategies. If you would like to see what was presented during the live update, I can suggest you go to the [event’s program](https://events.prestashop.com/prestashop-developer-conference/en/sessions) on the dedicated website.
```
Comment (coffrep) [inline content/news/posts/2023/prestashop-developer-conference-2023.md]:
```suggestion
During the day, presentations covered a wide array of topics, ranging from module development to new features of PrestaShop. The event also highlighted developer solutions, security concerns, and CI/CD deployment strategies. If you would like to see what was presented during the conference, I can suggest you go to the [event’s program](https://events.prestashop.com/prestashop-developer-conference/en/sessions) on the dedicated website.
```

File: content/news/posts/2023/prestashop-developer-conference-2023.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+Between sessions, attendees took breaks and were able to grab refreshments as well as coffee. It was the perfect time to catch up with some acquaintances, get to know new people, network to better know the community and how to get more involved with the project. Lunch was also a special treat as PrestaShop had organized a pickup buffet and participants were able to enjoy a number of catered foods, still being able to discuss with their peers.
+
+<img src="/assets/images/2023/12/psdevcon2.jpg" style="height:400px; margin: 20px auto;">
+
+At the end of the day, a special Q&A happened with [Eric Sénéchal](https://www.linkedin.com/in/senechaleric/), [Olivier Fontenelle](https://www.linkedin.com/in/olivierfontenelle/), [Clément Paillasse](https://www.linkedin.com/in/clementpaillasse/) (PrestaShop CPO) and Mathieu Ferment answering questions. Questions were open to everyone and discussions were very interesting, starting with a summary of the day, top of mind topics for Clément and Olivier. There was also the exciting announcement of FOP Day 2024 in Marseille, questions about more company related topics as well as next project versions, with a special emphasis on PrestaShop 9.
```
Comment (coffrep) [inline content/news/posts/2023/prestashop-developer-conference-2023.md]:
```suggestion
At the end of the day, a special Q&A happened with [Eric Sénéchal](https://www.linkedin.com/in/senechaleric/), [Olivier Fontenelle](https://www.linkedin.com/in/olivierfontenelle/), [Clément Paillasse](https://www.linkedin.com/in/clementpaillasse/) (PrestaShop CPO), Mathieu Ferment and Jonathan Lelievre answering questions. Questions were open to everyone and discussions were very interesting, starting with a summary of the day, top of mind topics for Clément and Olivier. There was also the exciting announcement of FOP Day 2024 in Marseille, questions about more company related topics as well as next project versions, with a special emphasis on PrestaShop 9.
```

**Review by boherm — APPROVED** [review]

**Review by coffrep — APPROVED** [review]
Hi @kpodemski I'm approving the post with some modifications. Hope they make sense.

**Pull request merged**